js/taxclock/tc-incomecalculator.js

```diff
@@ -28,6 +28,12 @@ var IncomeCalculator = function() {
   this.PERSONAL_INCOME_TAX_REVENUE = {{ site.income_tax_revenue }} * Math.pow(10,9);
   this.VAT_REVENUE = {{ site.vat_revenue }} * Math.pow(10,9);
 
+  // Housing Levy: https://kra.go.ke/images/publications/The-Finance-Act--2023.pdf#page=55
+  this.HOUSING_LEVY = {{ site.housing_levy }} * Math.pow(10,9);
+
+  // https://apps.wingubox.com/blog/category/updates/new-shif-rates-for-your-kenyan-payroll
+  this.SHIF = {{ site.shif }} * Math.pow(10,9);
```

Comment:
On the [site](https://taxclock.pesacheck.org/), it says:
>__What is your monthly income (before tax**)?__
>** This should be your income before tax but after deductions such as pension and health insurance.
>

Is SHIF different?

Reply:
@kilemensi SHIF is a state mandated health tax, that can be quite different from the standard health insurance providers such as Safety Wing. For SHIF, it does not matter if you have additional/alternative health insurance coverage, it will still be deducted from your gross salary as a tax item